packages/core-ethereum/src/indexer/types.ts

```diff
@@ -33,6 +33,8 @@ export type IndexerEvents =
   | `withdraw-hopr-${string}`
   | `withdraw-native-${string}`
   | `channel-updated-${string}`
+  | `on-provider-error-${string}`
```

Comment:
Please add a comment in the code (where appropriate) mentioning the mechanism of how the transactions are retried thanks to this.